types/fuzzy-search/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for fuzzy-search 2.1
+// Project: https://github.com/wouter2203/fuzzy-search#readme
+// Definitions by: Alex Deas <https://github.com/alex-deas>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class FuzzySearch<T> {
+    constructor(haystack: T[], keys?: string[], options?: FuzzySearch.Options);
+    search(needle: string): T[];
+}
+
+declare namespace FuzzySearch {
+    interface Options {
+        caseSensitive?: boolean;
+        sort?: boolean;
+    }
+}
+
+export default FuzzySearch;
```

Comment:
If looks like `require("fuzzy-search")` is itself a function, not an object with a `default` property. So this should be `export = FuzzySearch;`.  The import could be left alone if you add `"esModuleInterop": true` to "compilerOptions", or you could change it to `import FuzzySearch = require("fuzzy-search");`.

Reply:
This still hasn't changed? Should be `export = FuzzySearch;` to match the package's runtime behavior.